cluster_scripts/daisy_chain.sbatch

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Right, the cluster kills anything that runs for more than 24 hours, and they removed the head nodes so it's no longer possible to run a coordinator script. To get around the issue I made this script, which reruns the pipeline every 24 hours. Since the scripts don't try to regenerate files that already exist, running this script multiple times doesn't duplicate too much work